examples/bind/turing.amp.html

```diff
@@ -0,0 +1,154 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>amp-bind: Turing machine</title>
+  <link rel="canonical" href="amps.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Questrial" rel="stylesheet" type="text/css">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+
+  <style amp-custom>
+    .advance {
+      background-color: green;
+    }
+    .return {
+      background-color: red;
+    }
+    button {
+      border: solid 1px black;
+    }
+    span {
+      padding: 5px;
+      border: solid 1px gray;
+    }
+  </style>
+</head>
+
+<body>
+  <button onclick="AMP.toggleExperiment('amp-bind');window.location.href=window.location.href;">Toggle &lt;amp-bind&gt; experiment</button>
+
+  <h2>Turing machine: 8-bit counter</h2>
+
+  <h3>Tape</h3>
+  <p>
+    <span [class]="T.head == 8 ? T.state : ''" [text]="T.cells[8]">0</span>
+    <span [class]="T.head == 7 ? T.state : ''" [text]="T.cells[7]">0</span>
+    <span [class]="T.head == 6 ? T.state : ''" [text]="T.cells[6]">0</span>
+    <span [class]="T.head == 5 ? T.state : ''" [text]="T.cells[5]">0</span>
+    <span [class]="T.head == 4 ? T.state : ''" [text]="T.cells[4]">0</span>
+    <span [class]="T.head == 3 ? T.state : ''" [text]="T.cells[3]">0</span>
+    <span [class]="T.head == 2 ? T.state : ''" [text]="T.cells[2]">0</span>
+    <span [class]="T.head == 1 ? T.state : ''" [text]="T.cells[1]">0</span>
+    <span [class]="T.head == 0 ? T.state : ''" class="advance">*</span>
+  </p>
+
+  <h3 [text]="'Count: ' + (count || 0)">Count: 0</h3>
+
+  <!--
```

Comment:
Yea, just some things I noticed when writing this sample. Planning to turn them into an issue after some more thought.